crates/wasmtime/src/config.rs

```diff
@@ -793,9 +802,11 @@ impl Config {
     /// the flag type.
```

Comment:
This would need to be updated. Also where would the error occur now?

Reply:
Thanks for your review!
The error occurs at `Engine::new` now.